pkg/api/cmd/apparmor/ls_linux.go

```diff
@@ -0,0 +1,83 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package apparmor
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"text/tabwriter"
+	"text/template"
+
+	types "github.com/containerd/nerdctl/pkg/api/types/apparmor"
+	"github.com/containerd/nerdctl/pkg/apparmorutil"
```

Comment:
Just a question, no action requested :) :

I wonder, while we move backend logic into `pkg` side, **should we also gradually collocate those backend logic and the specific `util` package into one package**? Take `apparmor` as an example:

1. we will have `pkg/cmd/apparmor` (or any `pkg/.../apparmor` decieded) for the backend logic.
2. we have `pkg/apparmorutil` for main functions related to `apparmor`.

I guess most functions in `pkg/cmd/apparmor` will be just take and (post)process some `ApparmorOption` fields and then call a function in `pkg/apparmorutil`.

cc @Zheaoli @AkihiroSuda @fahedouch

Reply:
I think this `xxutil` should only contain some pure helper function. The most important logic shouble be completed in the `pkg/xxx/{cmd}`